tools/data/sthv1/encode_videos.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+cd ../
+python build_videos.py ../../data/sthv1/rawframes/ ../../data/sthv1/videos/ --fps 12 --level 2 --start-idx 1 --filename-tmpl '%05d'
+echo "Genearte videos"
```

Comment:
and here.

generate means create something completely new. here we are encoding images to videos---a kind of format transformation